addons/project_sms/upgrades/1.1/pre-migrate.py

```diff
@@ -0,0 +1,12 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+def migrate(cr, version):
```

Comment:
As it is not critical, would not do it (internal migrations are done for broken data to recover, which is not the case here). People can easily modify the rule using the UI.